src/style.css

```diff
@@ -0,0 +1,96 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  color: white;
+  font-size: 100px;
+  text-align: center;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  max-width: 1000px;
+}
+
+.block {
+  min-width: 300px;
+  height: 300px;
+  flex-basis: 33.33%;
```

Comment:
use media to fix this value on mobile and tablet (your blocks must stretch to all device-width)